src/components/bc-start.ts

```diff
@@ -1,15 +1,62 @@
-import {customElement} from 'lit/decorators.js';
-import {BitcoinConnectElement} from './BitcoinConnectElement';
-import {withTwind} from './twind/withTwind';
-import {html} from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { BitcoinConnectElement } from './BitcoinConnectElement';
+import { withTwind } from './twind/withTwind';
+import { html } from 'lit';
 import './internal/bci-button';
 import './bc-connector-list';
-import {classes} from './css/classes';
-import {disconnectSection} from './templates/disconnectSection';
+import { classes } from './css/classes';
+import { disconnectSection } from './templates/disconnectSection';
+import { fiat } from '@getalby/lightning-tools';
 
 // TODO: split up this component into disconnected and connected
 @customElement('bc-start')
 export class Start extends withTwind()(BitcoinConnectElement) {
+  @state()
+  _fiatBalance?: string;
+
+  @property({ type: Number })
+  override _balance?: number;
+
+  override connectedCallback() {
+    super.connectedCallback();
+
+
+    this.addEventListener('balance-set', () => {
+      this.fetchFiatBalance();
+    });
+  }
+
+
+  override updated(changedProperties: Map<PropertyKey, unknown>) {
+    super.updated(changedProperties);
+
+    if (changedProperties.has('_balance')) {
+
+      this.dispatchEvent(new Event('balance-set'));
+    }
+  }
+
+  private async fetchFiatBalance() {
+    if (!this._balance) {
+      return;
+    }
+
+    try {
+      const fiatBalance = await fiat.getFiatValue({
+        satoshi: this._balance,
+        currency: 'usd',
+      });
+
+      this._fiatBalance = fiatBalance.toLocaleString(undefined, {
+        style: 'currency',
+        currency: 'USD',
+      });
+    } catch (error) {
+      console.error('Error fetching fiat balance:', error);
+      this._fiatBalance = 'Error fetching fiat balance';
```

Comment:
Hmm, I don't think we should show this in the same variable - this will probably look bad if it happens.

Maybe we can just not show the fiat balance until it's successfully loaded, or we could use a skeleton loader there. So if it stays loading infinitely it's likely there was an issue fetching the fiat balance.